lib/Doctrine/ODM/MongoDB/Mapping/ClassMetadataFactory.php

```
@@ -207,6 +208,15 @@ protected function doLoadMetadata($class, $parent, $rootEntityFound, array $nonS
 
         $eventArgs = new LoadClassMetadataEventArgs($class, $this->dm);
         $this->evm->dispatchEvent(Events::loadClassMetadata, $eventArgs);
+
+        if ($class->isChangeTrackingNotify()) {
```

the CS build wants to invert the condition to use early exit, but in this case the code is more readable the way it is

---

I've decided to ignore the error. Offending lines will be gone eventually and so will be the suppression. Is this OK?